src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import entity.Deadline;
+import entity.Event;
+import entity.Task;
+import entity.Todo;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Main application driver class.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+  static List<Task> addedTasks = new ArrayList<Task>();
```

Comment:
Can consider renaming addedTasks to tasks to enhance readability

Reply:
Will consider definitely, thank you